### DISCUSSION TITLE
How can I change the database password?

### DISCUSSION THREAD

**Discussion by d-mal** [Kunzisoft/KeePassDX#1654] [Q&A] [answered]
Hi, 
I was trying to change the password to a .kdbx database in KeePassDX, and can't find a way to do it. 
Is it even possible to do that? And, if yes, then how?

Thanks for any help!

**Comment by J-Jamet** [comment]
Of course it's possible. First you need to open your database. You'll find the options in the database settings.

**Reply by shuvashish76** [reply, marked as answer]
Settings > Master key > Change master key
